libs/filabridge/include/private/filament/EngineEnums.h

```diff
@@ -43,7 +43,7 @@ namespace BindingPoints {
     constexpr uint8_t PER_RENDERABLE          = 1;    // uniforms/samplers updated per renderable
     constexpr uint8_t PER_RENDERABLE_BONES    = 2;    // bones data, per renderable
     constexpr uint8_t LIGHTS                  = 3;    // lights data array
-    constexpr uint8_t POST_PROCESS            = 4;    // samplers for the post process pass
+    constexpr uint8_t POST_PROCESS            = 4;    // deprecated
```

Comment:
I'm not sure how we want to handle this? Just delete the binding point and move `PER_MATERIAL_INSTANCE` down?

Reply:
At least rename it to `POST_PROCESS_DEPRECATED` so we're sure it's not used anywhere and won't be.

Reply:
Good idea